src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -211,6 +213,33 @@ unsigned long DrawSketchHandler::getCrosshairColor()
     return color;
 }
 
+bool DrawSketchHandler::distanceXYorPointOnObject(bool distanceXZeroYOne, int geoId, Sketcher::PointPos posId, double distance) {

```

Comment:
I cannot find any reason why a function declared in DrawSketchHandler.h would be defined in CommandCreateGeo.cpp. 

Either the code is generic at DrawSketchHandler level or it belongs elsewhere (helper class, intermediate derived class from DrawSketchHandler,...).

What are you trying to achieve?

Reply:
I reviewed this question and found out that I put the declaration here because this function is used by many DrawSketchHandlers.
To the question why would it be defined in CommandCreateGeo.cpp, I just replicated how other functions are done : setCrosshairColor, getCrosshairColor.
Though it might be better if it is implemented as functions in "/* helper functions" ?